playbooks/roles/deploy-osh/templates/nova.yaml.j2

```diff
@@ -6,4 +6,50 @@ conf:
       user: {{ ses_cluster_configuration['cinder']['rbd_store_user'] }}
       keyring: {{ suse_ses_cinder_pool_key | default(ceph_admin_keyring_key) }}
       secret_uuid: {{ libvirt_ceph_cinder_secret_uuid }}
-{{ lookup('template','files/common-images-overrides.yml') | from_yaml | to_nice_yaml(indent=2) }}
+{% if developer_mode %}
+  apache2:
+    binary: apache2ctl
+    start_parameters: -DFOREGROUND -k start
+    site_dir: /etc/apache2/vhosts.d
+    conf_dir: /etc/apache2/conf.d
```

Comment:
I think you need version module here as well

Reply:
that is what I thougth but someone this does not fail...

I do not understand apache sometimes